frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -978,15 +979,24 @@ export default ({
         }
       }
     },
-    updateReadUntilMessageHash ({ messageHash, createdHeight, forceUpdate = false }) {
+    updateReadUntilMessageHash ({
+      messageHash,
+      createdHeight,
+      // 'forceUpdate' flag here is for the rare case where the 'readUntil' value needs to be set to the msg with lower 'createdHeight'.
+      // eg. when the latest message is deleted. (reference: https://github.com/okTurtles/group-income/issues/2729)
+      forceUpdate = false,
+      // 'Mark unread' feature allows user to set 'currentChatRoomReadUntil' to the message they want.
+      // So if user has used this functionality at least once, the chatroom should stop auto-updating the 'readUntil' data in the store (eg. while scrolling),
+      // So that the message that has been marked unread is kept until user leaves the chatroom and re-enter in the future.
+      // Hence, setting below flag to 'true' by default.
+      noUpdateWhenMarkUnreadIsUsed = true
```

Comment:
I checked the codebase and this `noUpdateWhenMarkUnreadIsUsed` flag isn't used anywhere... so why have it?

Reply:
Had an idea for this but it doesn't seem to be needed. will remove it.

Reply:
Removed it.